lib/modules/dosomething/dosomething_reportback/includes/ReportbackItemTransformer.php

```diff
@@ -18,6 +18,7 @@ public function index($parameters) {
     }
     catch (Exception $error) {
       // @TODO: Potentially log error to watchdog.
+      http_response_code('404');
```

Comment:
How would we get a 404 on the index endpoint? If we're throwing an error here, shouldn't we return an `error` object instead of `data` (2 lines below?)